replay_build_scripts/common.mjs

```diff
@@ -154,11 +148,55 @@ export function updateChromiumRepo() {
   const branch = process.env["BUILDKITE_BRANCH"];
   updateRepo(chromium, `origin/${branch}`);
 
+  const deps = getChromiumDeps();
+
+  syncRepo(path.join(chromium, "v8"), deps.v8);
```

Comment:
Can we make sure that we only have one hardcoded list of *all* our repos?
Can't we just iterate over `deps` instead of listing them again?

Reply:
I don't understand - we do have a single hardcoded list of our repos.  That's `DEPS`.

Reply:
I'm talking about the code. You hardcoded the list in code in 2 places 😅
* here: https://github.com/replayio/chromium/pull/1107/files#diff-a8e7da11c4aab103c43d02a564df77e190845a15204731fd31db50730f261e47R153
* here: https://github.com/replayio/chromium/pull/1107/files#diff-a8e7da11c4aab103c43d02a564df77e190845a15204731fd31db50730f261e47R172